dark_evaluation.py

```diff
@@ -27,23 +27,22 @@
     digicam_geometry = geometry.generate_geometry_from_camera(camera=digicam)
 
     # Define the event stream
-    data_stream = event_stream(file_list=file_list, expert_mode=True, camera_geometry=digicam_geometry)
+    data_stream = event_stream(file_list=file_list, expert_mode=True, camera_geometry=digicam_geometry, camera=digicam)
+    data_stream = r0.fill_trigger_input_7(data_stream)
```

Comment:
this should be done after "setting patches to zero"

Reply:
thx